2-positioning/style.css

```diff
@@ -0,0 +1,25 @@
+header {
+    border: 3px solid greenyellow;
+    background-color: rgb(0, 0, 0);
+    color: white;
+    position: sticky;
+    width: 100%;
+}
+nav{
+    position: absolute;
+    border: 3px  blue solid;
+    width: 200px;
+}
+section{
+    position: static;
+    border: 3px  red solid;
+    margin-left: 200px;
+}
+footer {
+    border: 3px  orangered solid;
+    background-color: rgb(0, 0, 0);
+    color: white;
+    position: fixed;
+    width: 100%;
+    bottom:0;
+}
```

Comment:
Please reorder these properties and the declarations according to the alphabet (a-z).